server/utils/sessionStatusTags.ts

```diff
@@ -0,0 +1,18 @@
+export default {
+  notScheduled: {
+    text: 'NOT SCHEDULED',
```

Comment:
I think we can get rid of this file now?

Reply:
Oops, sorry